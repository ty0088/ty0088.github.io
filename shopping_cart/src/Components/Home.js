import React from 'react';
import '../Style/style.css';
import homeImg from '../Images/home.jpg'

const Home = () => {
    return (
        <div>
            <p>Welcome to Cats'R'Us Store.</p>
            <p>Cats'R'Us Store is the leading cat store for all kinds of cats.</p>
            <p>C'mon, let's Purrrr!</p>
            <img src={homeImg} alt="cat sitting on a stool"></img>
        </div>
    );
};

export default Home;