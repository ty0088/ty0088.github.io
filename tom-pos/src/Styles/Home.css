#nav-home {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 460px 40px;
}

.link-container {
    display: flex;
    flex-direction: row;
    justify-content: center;
    align-items: center;
    flex-wrap: wrap;
}

.home-link {
    display: flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    height: 150px;
    width: 150px;
    border: 1px solid black;
    border-radius: 5px;
    margin: 10px;
    color: black;
}

.home-link:visited {
    color: black;
}

.home-link:hover {
    font-weight: 900;
    border: 2px solid black;
}

.nav-footer {
    display: flex;
    align-items: center;
    justify-content: space-around;
}

.nav-footer > button {
    height: 25px;
    width: 80px;
}


.foot-link {
    color: black;
    text-decoration: none;
}

.foot-link:visited {
    color: black;
}

.foot-link:hover {
    font-weight: 900;
}