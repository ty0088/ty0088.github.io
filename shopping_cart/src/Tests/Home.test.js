import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Components/Home';

describe("Home component", () => {
    it("Snapshot", () => {
      const { container } = render(<Home />);
      expect(container).toMatchSnapshot();
    });
});