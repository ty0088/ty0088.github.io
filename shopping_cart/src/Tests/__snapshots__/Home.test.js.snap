// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Snapshot 1`] = `
<div>
  <div
    class="flex-column-center"
  >
    <p>
      Welcome to Cats'R'Us Store.
    </p>
    <p>
      Cats'R'Us Store is the leading cat store for all kinds of cats.
    </p>
    <p>
      C'mon, let's Purrrr!
    </p>
    <br />
    <img
      alt="cat sitting on a stool"
      src="home.jpg"
    />
  </div>
</div>
`;
